keyboards/ginny/ginny.c

```diff
@@ -0,0 +1,72 @@
+#include QMK_KEYBOARD_H
+
+bool i2c_initialized = 0;
+i2c_status_t mcp23018_status = 0x20;
+
+void matrix_init_kb(void) {
+    // (tied to Vcc for hardware convenience)
```

Comment:
Just a note that we do have pin functions in qmk. It won't be as compact, but it'd be more readable. for example:

```C
setPinInputHigh(B0);
setPinInputHigh(B4);
setPinInputHigh(B5);
setPinInputHigh(B6);
setPinInputHigh(B7);
```
For the portB pins.

Full list here: https://beta.docs.qmk.fm/for-makers-and-modders/internals_gpio_control#functions